docs/user/how-to/run-cli.rst

```diff
@@ -0,0 +1,80 @@
+Control the Worker via the CLI
+==============================
+
+The worker comes with a minimal CLI client for basic control. It should be noted that this is 
+a test/development/debugging tool and not meant for production!
```

Comment:
We use the CLI for the dockerfile. In future will the server be spun up any differently?

Reply:
Good point, I forgot that this page refers only to the `blueapi controller` CLI. Have moved the stuff about ActiveMQ and the `serve` command to a new page called "Quckstart Guide".